frontend/ui/elements/screen_notification_menu_table.lua

```diff
@@ -0,0 +1,175 @@
+    if toggle and toggle==true then
+        return {
+            text = _("Expert notification settings"),
+            help_text =_("Here you can set the sources of popup notifications on a fine grain scale."),
+            sub_item_table = {
+                {
+                text = _("Bottom menu icons"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_BOTTOM_MENU_ICON)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_BOTTOM_MENU_ICON)
+                end,
+                },
+                {
+                text = _("Bottom menu toggles"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_BOTTOM_MENU_TOGGLE)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_BOTTOM_MENU_TOGGLE)
+                end,
+                },
+                {
+                text = _("Bottom menu fine tuning"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_BOTTOM_MENU_FINE)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_BOTTOM_MENU_FINE)
+                end,
+                },
+                {
+                text = _("Bottom menu three dots"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_BOTTOM_MENU_MORE)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_BOTTOM_MENU_MORE)
+                end,
+                },
+                {
+                text = _("Gestures and profiles"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_DISPATCHER)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_DISPATCHER)
+                end,
+                },
+                {
+                text = _("Other sources"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_OTHER)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_OTHER)
+                end,
+                },
+            },
+        }
+    end
+    return ""
+end
+
+return {
+    text = _("Notification level"),
+    help_text = _([[KOReader may show notification popups at top of screen on various occasions.
+You can decide here which kind of notifications you want displayed.]]),
+    checked_func = function()
+        local value = G_reader_settings:readSetting("notification_sources_to_show_mask") or Notification.SOURCE_DEFAULT
+        return  value ~= 0
+    end,
+    sub_item_table = {
+        {
+        text = _("No popups"),
+        help_text = _("No popups are shown at all."),
```

Comment:
```
No notification.
No notification popups will be shown.
```
Notification is our internal technical name, but let's use it also for the UI. Other things can "pop up" :)
But ok to use "Notification popups" in help_text to be more verbose and give this alternative explanation of what it is.